brave/slack.py

```diff
@@ -18,10 +18,7 @@ def notify_channel_internal(api_key, channel, message):
 def notify_user_internal(api_key, user_id, message):
     client = WebClient(token=api_key)
     try:
-        response = client.conversations_open(users=[user_id])
-        response = client.chat_postMessage(
-            channel=response['channel']['id'],
-            text=message)
+        _ = client.chat_postMessage(channel=user_id, text=message)
```

Comment:
FYI, this change is bec we'd like the GH - slack mapping to use slack usernames (not user IDs). So a sample map file would be
```json
{
    "bbondy": "@bbondy",
}
```
conversations_open insists on IDs, while removing it seems to work with user friendly usernames.